geometry_msgs/Pose goal
---
bool plan_found

